Wed Sep  1 13:42:40 CEST 1999	Marc Espie <espie@cvs.openbsd.org>
	* libiberty.h (basename):  OpenBSD has a correct prototype.
	(xrealloc):  Remove outdated comment.

--- include/libiberty.h.orig	Wed Sep  1 13:40:21 1999
+++ include/libiberty.h	Wed Sep  1 13:41:51 1999
@@ -36,7 +36,7 @@ extern char **dupargv PARAMS ((char **))
    across different systems, sometimes as "char *" and sometimes as
    "const char *" */
 
-#if defined (__GNU_LIBRARY__ ) || defined (__linux__) || defined (__FreeBSD__) || defined (__CYGWIN__) || defined (__CYGWIN32__)
+#if defined (__GNU_LIBRARY__ ) || defined (__linux__) || defined (__FreeBSD__) || defined (__OpenBSD__) || defined (__CYGWIN__) || defined (__CYGWIN32__)
 extern char *basename PARAMS ((const char *));
 #else
 extern char *basename ();
@@ -128,10 +128,7 @@ extern void xmalloc_set_program_name PAR
 #endif
 extern PTR xmalloc PARAMS ((size_t));
 
-/* Reallocate memory without fail.  This works like xmalloc.
-
-   FIXME: We do not declare the parameter types for the same reason as
-   xmalloc.  */
+/* Reallocate memory without fail.  This works like xmalloc.  */
 
 extern PTR xrealloc PARAMS ((PTR, size_t));
 
